src/main/resources/vm/script/datamanagement/downloadFunctions.vm

```diff
@@ -127,6 +176,18 @@ function downloadURI {
         downloadGirderFile ${URI}
         validateDownload "Cannot download Girder file"
     fi
+
+    if [[ ${URI_LOWER} == gfal:/* ]] 
+    then
+        mountGfal ${URI}
+        validateDownload "Cannot download gfal file"
```

Comment:
I am not sure the validateDownload will work, as it tests `$?`
In case on `mountGfal`, the last command is an `echo`, which should always success and could hide a error. But I am not sure how this works like that in a case of method call. Have you tested it ?